### PR TITLE
CSS with styling and implemented some server side tests

### PR DESCRIPTION
**Pull request by JeffreyStocker**
***** NOTE: MUST HAVE ENVIRONMENTAL VARIABLES SET!!!! OR TESTS WILL FAIL!!!!!
I recommend implementing a module that can set env variables

### CODE REVIEW

File: components/Navigation.js [hunk 1]
```diff
@@ -36,17 +59,60 @@ class Navigation extends React.Component {
...
+            autoCloseWhenOffScreen
+            canAutoPosition={true}
+            animated={true}
+          >
+            <Menu>
+              <MenuItem primaryText="Sort by Date" onClick={(evt) => this.navButtonClick('navBtnGraph')} />
+              <MenuItem primaryText="Sort by Relative to Release Date" onClick={(evt) => this.navButtonClick('navBtnGraphRelative')} />
+            </Menu>
+          </Popover>
+        {/* <RaisedButton
```
Comment (janetlee) [inline components/Navigation.js:91]: What is this comment stuff doing in the curly braces? Not a complaint, just a question

File: containers/MovieDetail.jsx [hunk 2]
```diff
@@ -36,12 +36,13 @@ function MovieDetail(props) {
   } else if (mainView === 'map') {
     returnDom = null;
   } else if (mainView === 'locationSentiment') {
-    returnDom = null;
+    returnDom = <Location />;
```
Comment (janetlee) [inline containers/MovieDetail.jsx:39]: 👍

File: specs/server/test_locationRoute.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const { expect } = require('chai');
+const request = require('supertest');
+var server = require('../../app.js');
+
+var dmdbKey = process.env.API_KEY || null;
+
+describe ('information from sentiment route with given latitude and longatude', () => {
```
Comment (janetlee) [inline specs/server/test_locationRoute.js:7]: "Longitude" 😆

File: specs/server/test_tmdb.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    })
+
+    it ('should expect data to contain: page, total_results, total_pages, and results', () =>{
+      expect(data.page).to.exist
+      expect(data.total_results).to.exist
+      expect(data.total_pages).to.exist
+      expect(data.results).to.exist
+    })
+    it ('results should contain a array of movies', () =>{
+    //   { vote_count : 7176,
```
Comment (janetlee) [inline specs/server/test_tmdb.js:29]: I think this should be a test file in the repo and not comment text here.

File: specs/server/test_tmdb.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+      'tagline',
+      'title',
+      'video',
+      'vote_average',
+      'vote_count' ]
+      expect(data).to.be.a('object');
+      expect(Object.keys(data)).to.eql(expected);
+    })
+
+        // { adult: false,
```
Comment (janetlee) [inline specs/server/test_tmdb.js:104]: See comment above. It's not good to have this hard code in here b/c someone may interpret this to mean it's a fixed set of values returning from the API. It's actually the sample shape of data. Namely the votecount could change or the total revenue.

**Comment by janetlee** [comment]
LGTM. Nothing here is a showstopper. Asking @JeffreyStocker  to read the comments though re: the sample data commented out in the files.

File: components/Navigation.js (hunk 1 above)
Comment (JeffreyStocker) [reply components/Navigation.js:91]: its how you comment inside the render for jsx

File: containers/MovieDetail.jsx (hunk 2 above)
Comment (JeffreyStocker) [reply containers/MovieDetail.jsx:39]: got to love rebasing

File: specs/server/test_locationRoute.js (hunk 3 above)
Comment (JeffreyStocker) [reply specs/server/test_locationRoute.js:7]: hahaha

File: specs/server/test_tmdb.js (hunk 4 above)
Comment (JeffreyStocker) [reply specs/server/test_tmdb.js:29]: yea i agree, i was trying to avoid creating too many files

File: specs/server/test_tmdb.js (hunk 5 above)
Comment (JeffreyStocker) [reply specs/server/test_tmdb.js:104]: i was using to see what the data looked like, yea should be deleted now the test is created

**Comment by JeffreyStocker** [comment]
thank!!

**Pull request merged**